src/ds++/DracoStrings.cc

```diff
@@ -0,0 +1,175 @@
+//----------------------------------*-C++-*----------------------------------//
+/*!
+ * \file   ds++/DracoStrings.cc
+ * \author Kelly G. Thompson <kgt@lanl.gov
+ * \date   Wednesday, Aug 23, 2017, 12:48 pm
+ * \brief  Enscapulates common string manipulations (implementation).
+ * \note   Copyright (C) 2017 Los Alamos National Security, LLC.
+ *         All rights reserved. */
+//---------------------------------------------------------------------------//
+
+#include "DracoStrings.hh"
+#include <fstream>
+#include <iostream>
+
+namespace rtt_dsxx {
+
+//----------------------------------------------------------------------------//
+// Definitions for fully specialized template functions
+//----------------------------------------------------------------------------//
+
+template <> auto parse_number<int>(std::string const &str) -> int {
```

Comment:
Do we want any sort of error- or input-checking here? For instance, I think stoi could read something like "123abc" and return 123-- is this a valid use case?